internal/report/report.go

```diff
@@ -89,8 +89,12 @@ func newReport(results Results) (*reportStatus, bool) {
 		reportMutatorType(m, rep)
 	}
 	if !rep.isDryRun() {
-		rep.tEfficacy = float64(rep.killed) / float64(rep.killed+rep.lived) * 100
-		rep.mCovered = float64(rep.killed+rep.lived) / float64(rep.killed+rep.lived+rep.notCovered) * 100
+		if rep.killed > 0 {
+			rep.tEfficacy = float64(rep.killed) / float64(rep.killed+rep.lived) * 100
+		}
+		if rep.killed+rep.lived > 0 {
+			rep.mCovered = float64(rep.killed+rep.lived) / float64(rep.killed+rep.lived+rep.notCovered) * 100
+		}
 	} else {
 		rep.mCovered = float64(rep.runnable) / float64(rep.runnable+rep.notCovered) * 100
```

Comment:
Shouldn't we check here as well if it is a divide by 0?

Reply:
On 96 it's handled with if statement on line 95. I will check also line 95 for division by zero in this issue, what do you think?